api/fetch_good_first_issue.go

```diff
@@ -67,11 +68,15 @@ func GoodFirstIssue(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// Take current unix nano as seed.
-	rnd := rand.New(rand.NewSource(time.Now().UnixNano()))
-	index := rnd.Intn(len(issues))
+	content, err := json.Marshal(issues)
+	if err != nil {
+		log.Fatalf("Invalid issues: %s", err)
+	}
+
+	// Cache content up to 600 (10 mins)
+	w.Header().Add("Cache-Control", "s-maxage=600")
```

Comment:
Better to cache on server side.

Reply:
Yep, it does cache on the server-side. Vercel will the static on edge so our next API call will get the cache instead of sending new API calls.